yt_dlp/YoutubeDL.py

```diff
@@ -702,6 +713,14 @@ def check_deprecated(param, option, suggestion):
         for msg in self.params.get('_deprecation_warnings', []):
             self.deprecated_feature(msg)
 
+        impersonate_target = self.params.get('impersonate')
+        if impersonate_target is not None:
+            # This assumes that all handlers that support impersonation subclass ImpersonateRequestHandler
```

Comment:
Comment should be in impersonate_target_available

Reply:
good catch